hexrd/ui/main_window.py

```diff
@@ -377,7 +377,7 @@ def images_loaded(self, enabled=True):
         self.update_color_map_bounds()
         self.update_hedm_enable_states()
         self.color_map_editor.reset_range()
-        self.image_mode_widget.reset_masking(enabled)
+        self.image_mode_widget.reset_masking()
```

Comment:
So this means masking should be disabled every time new images are loaded, then?

Reply:
Yes, unless there is a reason to default to threshold masking on for every new image?

Also, thinking about this a little more no matter whatever the default should be, I am thinking the masking may not be being handled properly. Right now when a threshold mask is applied a copy of the imageseries is created and handled as an array. Should this be handled more like the aggregated images where we access the copy of the ProcessedImageSeries when thresholding is toggle on? Or should the original imageseries be getting updated as the masking is changed (and if so should it happen with each change or should the user have to "accept" the current settings before the update)?